openedx/features/enterprise_support/utils.py

```diff
@@ -300,18 +303,19 @@ def get_enterprise_learner_portals(request):
     # Prevent a circular import.
     from openedx.features.enterprise_support.api import enterprise_enabled
 
-    if enterprise_enabled():
+    user = request.user
+    # Only cache this if a learner is authenticated (AnonymousUser exists and should not be tracked)
+    if enterprise_enabled() and enterprise_header_links.is_enabled() and user and user.id:
```

Comment:
should `enterprise_header_links` be capitalized to match the variable above?

Reply:
Yes bad refactor here, had it lower case at first and really dropped the ball before pushing up changes.